src/components/dapp-staking/stake-manage/StakeForm.vue

```diff
@@ -64,22 +64,21 @@
       </div>
 
       <div class="separator" />
-      <!-- Todo: update logic -->
       <SpeedConfigurationV2
         :gas-cost="nativeTipPrice"
         :selected-gas="selectedTip"
         :set-selected-gas="setSelectedTip"
       />
 
-      <div
-        v-if="errMsg && currentAccount"
-        class="row--box-error"
-        :class="isRequiredCheck && 'box--margin-adjuster'"
-      >
+      <div v-if="errMsg && currentAccount" class="row--box-error">
         <span class="color--white"> {{ $t(errMsg) }}</span>
       </div>
       <div class="wrapper__row--button" :class="!errMsg && 'btn-margin-adjuster'">
-        <astar-button class="btn-size--confirm" :disabled="errMsg || !Number(amount)">
+        <astar-button
```

Comment:
I see inconsistent component name format (`kebab-case` and `PascalCase`).

Can we unify them into one and add format checks?

Reply:
Ok, Let's use kebab-case from now on.
CC: @bobo-k2 @sirius651 

> Can we unify them into one and add format checks?
@hoonsubin I'll add this on backlog, I will create another PR for this later.